### ISSUE TITLE
The computer encountered a blue screen while running the script "run_localGPT.py".

### ISSUE THREAD

**Issue by zerodayZHOUXUTIAN** [PromtEngineer/localGPT#107]
The error code shown on the blue screen is "IRQL_NOT_LESS_OR_EQUAL".

![image](https://github.com/PromtEngineer/localGPT/assets/30335316/25120d37-e8c7-4633-80e3-6506e2244b4b)

**Comment by jackfood** [comment]
Try to increase your pagefile for RAM.

**Comment by oppenheimer-** [comment]
yeah, the swapping goes hard. but it doesn't seem to work. the whole system turns unbearably slow and i don't see any progress when the model is loading. Got 32Mb and already tried the 8bit flag, too. But without success.

**Comment by zerodayZHOUXUTIAN** [comment]
> 
 It still doesn't work.Is it impossible to run a project with 16GB of memory ?
![image](https://github.com/PromtEngineer/localGPT/assets/30335316/4ac6f8c3-09a7-4f82-b3e7-f895f187ca1c)

**Comment by zerodayZHOUXUTIAN** [comment]
> Try to increase your pagefile for RAM.
It still doesn't work.Is it impossible to run a project with 16GB of memory ?
![image](https://github.com/PromtEngineer/localGPT/assets/30335316/989c65be-110c-4c2b-8013-ca384d2ca4b8)

**Comment by zerodayZHOUXUTIAN** [comment]
> you may not have enough RAM to run it. happened to me as well, except for the blue screen

It still doesn't work.Is it impossible to run a project with 16GB of memory ?
![image](https://github.com/PromtEngineer/localGPT/assets/30335316/8cbb7bfc-3429-493f-bf77-37fa8cd68fe9)

**Comment by zerodayZHOUXUTIAN** [comment]
> Try to increase your pagefile for RAM.

It still doesn't work.Is it impossible to run a project with 16GB of memory ?

**Comment by mfarooq0001** [comment]
> 

Yes, you can run it with 16GB RAM.
